Marlin/ultralcd.cpp

```diff
@@ -1523,6 +1503,59 @@ void kill_screen(const char* lcd_msg) {
     lcd_return_to_status();
   }
 
+  #if ENABLED(FILAMENT_LOADUNLOAD_SUPPORT)
+    void lcd_filament_load_unload_menu() {
+      START_MENU();
+      MENU_BACK(MSG_PREPARE);
+      // Load filament
+      #if E_STEPPERS == 1
+        if (!thermalManager.targetTooColdToExtrude(active_extruder))
+          MENU_ITEM(gcode, MSG_FILAMENTLOAD, PSTR("M701"));
+      #else
+        if (!thermalManager.targetTooColdToExtrude(0))
+          MENU_ITEM(gcode, MSG_FILAMENTLOAD1, PSTR("M701 T0"));
+        if (!thermalManager.targetTooColdToExtrude(1))
+          MENU_ITEM(gcode, MSG_FILAMENTLOAD2, PSTR("M701 T1"));
+        #if E_STEPPERS > 2
+          if (!thermalManager.targetTooColdToExtrude(2))
+            MENU_ITEM(gcode, MSG_FILAMENTLOAD3, PSTR("M701 T2"));
+          #if E_STEPPERS > 3
+            if (!thermalManager.targetTooColdToExtrude(3))
+              MENU_ITEM(gcode, MSG_FILAMENTLOAD4, PSTR("M701 T3"));
+            #if E_STEPPERS > 4
+              if (!thermalManager.targetTooColdToExtrude(4))
+                MENU_ITEM(gcode, MSG_FILAMENTLOAD5, PSTR("M701 T4"));
+            #endif // E_STEPPERS > 4
+          #endif // E_STEPPERS > 3
+        #endif // E_STEPPERS > 2
+      #endif // E_STEPPERS == 1
+
+      // Unload filament
+      #if E_STEPPERS == 1
+        if (!thermalManager.targetTooColdToExtrude(active_extruder))
+          MENU_ITEM(gcode, MSG_FILAMENTUNLOAD, PSTR("M702"));
+      #else
+        if (!thermalManager.targetTooColdToExtrude(0))
+          MENU_ITEM(gcode, MSG_FILAMENTUNLOAD1, PSTR("M702 T0"));
```

Comment:
Please use `MSG_FILAMENT_UNLOAD MSG_E0` here. Then there is no need for multiple redundant `MSG_FILAMENTUNLOADn` defines.